src/templates/case-study.js

```diff
@@ -0,0 +1,174 @@
+            <div className={getBlocksWith('__button-group')}>
+              <Link
+                className={cx('btn', 'btn--white', 'btn--large', 'back-to-list')}
+                to={'/case-studies/'}
+              >
+                <svg
+                  xmlns="http://www.w3.org/2000/svg"
+                  width="16"
+                  height="16"
+                  viewBox="0 0 16 16"
+                  fill="none"
+                >
+                  <path
+                    fillRule="evenodd"
+                    clipRule="evenodd"
+                    d="M6.58866 3.66897C6.95812 3.25846 6.92484 2.62617 6.51433 2.25671C6.10382 1.88726 5.47153 1.92053 5.10207 2.33104L0 8.00001L5.10207 13.669C5.47153 14.0795 6.10382 14.1128 6.51433 13.7433C6.92484 13.3738 6.95812 12.7416 6.58866 12.331L3.59072 9H14C14.5523 9 15 8.55228 15 8C15 7.44772 14.5523 7 14 7H3.59073L6.58866 3.66897Z"
+                    fill="white"
+                  />
+                </svg>
+                Back to Case Studies
+              </Link>
+              <Link
+                className={cx('btn', 'btn--white', 'btn--large', 'download-pdf')}
+                onClick={() => {}}
+              >
+                <svg
+                  xmlns="http://www.w3.org/2000/svg"
+                  width="16"
+                  height="16"
+                  viewBox="0 0 16 16"
+                  fill="none"
+                >
+                  <path
+                    d="M15 11V13C15 13.5304 14.7893 14.0391 14.4142 14.4142C14.0391 14.7893 13.5304 15 13 15H3C2.46957 15 1.96086 14.7893 1.58579 14.4142C1.21071 14.0391 1 13.5304 1 13V11"
+                    stroke="white"
+                    strokeWidth="2"
+                    strokeLinecap="round"
+                    strokeLinejoin="round"
+                  />
+                  <path
+                    d="M5 8L8 11L11 8"
+                    stroke="white"
+                    strokeWidth="2"
+                    strokeLinecap="round"
+                    strokeLinejoin="round"
+                  />
+                  <path
+                    d="M8 10V1"
+                    stroke="white"
+                    strokeWidth="2"
+                    strokeLinecap="round"
+                    strokeLinejoin="round"
+                  />
+                </svg>
+                Download PDF version
+              </Link>
+            </div>
+          </div>
+        </div>
+
+        <div className={getBlocksWith('__case-body')}>
+          <div className={getBlocksWith('__case-columns')}>
+            <div className={getBlocksWith('__challenges')}>
+              <div className={getBlocksWith('__challenges--body')}>
+                {challenges?.raw && renderRichText(challenges, options)}
+              </div>
+              <div className={getBlocksWith('__download-btn')}>
+                <button className={cx('btn', 'btn--primary', 'btn--large')} onClick={() => {}}>
```

Comment:
Let's use `noop` from lodash
```suggestion
                <button className={cx('btn', 'btn--primary', 'btn--large')} onClick={noop}>
```

Reply:
Fixed